Zenith-Fresh/src/app/api/system-monitor/route.js

```
@@ -0,0 +1,320 @@
+    // Update metrics before responding
+    updateMetrics();
+    
+    switch (endpoint) {
+      case 'overview':
+        return NextResponse.json(getDashboardMetrics());
+        
+      case 'metrics':
+        return NextResponse.json(systemMetrics);
+        
+      case 'history':
+        const hours = parseInt(url.searchParams.get('hours')) || 1;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)